nav2_behavior_tree/include/nav2_behavior_tree/bt_action_node.hpp

```diff
@@ -272,18 +316,38 @@ class BtActionNode : public BT::ActionNodeBase
         }
       };
 
-    auto future_goal_handle = action_client_->async_send_goal(goal_, send_goal_options);
+    goal_handle_.reset();
```

Comment:
I don't think this reset is required, its about to be overridden which implicitly destructs the old object